kibicara/kibicara_mda.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2020 by Maike <maike@systemli.org>
+#
+# SPDX-License-Identifier: 0BSD
+
+import email.parser
+from email.policy import default
+import email.message
+import sys
+import re
+import requests
+from logging import getLogger
+from kibicara.model import Hood
+from kibicara.platforms.email.model import Email
+import argparse
+from asyncio import run
+from fastapi import status
+from ormantic import NoMatch
+
+
+async def async_main(mail=None, hood_name=None):
```

Comment:
i rather keep the main code clear and simple, than provide parameters and code-paths only for testing. such mockups should be on the test side.